processor/eval/ledger_for_evaluator_test.go

```diff
@@ -551,11 +543,220 @@ func TestLedgerForEvaluatorAppCreatorMultiple(t *testing.T) {
 	}
 }
 
+// Test the functionality of `func (l LedgerForEvaluator) LookupKv()`.
+// This is done by handing off a pointer to Struct `processor/eval/ledger_for_evaluator.go::LedgerForEvaluator`
+// in the `CompareAppBoxesAgainstLedger()` assertions function which then asserts using `LookupKv()`
+func TestLedgerForEvaluatorLookupKv(t *testing.T) {
```

Comment:
Yes. While the overall scenarios are identical, this one checks that the local-ledger is getting the expected info about boxes, while the `postgres_intgration_test.go` actually queries indexer's DB after the box information ought to have been ingested to verify that everything works. You can see this because we have:
* `CompareAppBoxesAgainstLedger()` used to validate here
* `CompareAppBoxesAgainstDB()` used to validate in `postgres_integration_test.go`

There might be a way to do everything from the integration test (and in some sense, the integration is "good nuf" already because it subsumes this test), but since we have a lot of examples of testing separate components of indexer separately, and I believe that too many tests is much preferred over too little, I'd like to keep this test.

Reply:
I guess it's also possible to refactor in a way that runs the same logic but receives function pointer for the `CompareXYZ()` function. But that seems like a bit of overkill...